### ISSUE TITLE
Error in the documention of the Hessian class?

### ISSUE THREAD

**Issue by Alexander-Serov** [pbrod/numdifftools#51]
In the Hessian class definition I read
```
Returns
    -------
    hess : ndarray
       array of partial second derivatives, Hessian
    """, extra_note="""
    Computes the Hessian according to method as:
    'forward' :eq:`7`, 'central' :eq:`9` and 'complex' :eq:`10`:

    .. math::
        \quad ((f(x + d_j e_j + d_k e_k) - f(x + d_j e_j))) / (d_j d_k)
        :label: 7
```

The second term in this formula is non-symmetric towards `d_j <--> d_k`  substitution, which suggests it is wrong. I guess the term should just read `f(x)`.

I found this typo (?) while investigating for the reasons of weird Hessian results (negaitve Hessian determinant in a minimum), which I will post as a separate issue once I make sure it's not a problem on my side.

**Comment by pbrod** [comment]
Thanks for spotting this typo.  Actually it is computed as:
((f(x + d_j e_j + d_k e_k)  +f(x) - f(x + d_j e_j) - f(x + d_k e_k))) / (d_j d_k)

**Comment by pbrod** [comment]
Fixed in commit [f59a7d530](https://github.com/pbrod/numdifftools/commit/f59a7d530b3fea89a76d531c201bfd2c4a3195e4)
